lib/go-tc/servers.go

```diff
@@ -1101,6 +1105,70 @@ func (s *ServerV40) ToServerV2FromV4() (ServerNullableV2, error) {
 	return legacyServer, nil
 }
 
+type ServerUpdateStatusV4 ServerUpdateStatusV40
+
+type ServerUpdateStatusV40 struct {
+	HostName             string    `json:"host_name"`
+	UpdatePending        bool      `json:"upd_pending"`
+	RevalPending         bool      `json:"reval_pending"`
+	UseRevalPending      bool      `json:"use_reval_pending"`
+	HostId               int       `json:"host_id"`
+	Status               string    `json:"status"`
+	ParentPending        bool      `json:"parent_pending"`
+	ParentRevalPending   bool      `json:"parent_reval_pending"`
+	ConfigUpdateTime     time.Time `json:"config_update_time"`
+	ConfigApplyTime      time.Time `json:"config_apply_time"`
+	RevalidateUpdateTime time.Time `json:"revalidate_update_time"`
+	RevalidateApplyTime  time.Time `json:"revalidate_apply_time"`
+}
+
+// MarshalJSON is a custom implementation to ensure the date format returned is a RFC3339 string (and not RFC3339Nano or something else)
```

Comment:
Fixed. When testing I would see that the time was truncated, say with the `epoch` value T00:00:00 rather than say T00:00:00.00000. I wanted to enforce the Nano precision explicitly rather than implicitly.